resolwe_bio/tools/spikein_pairwise.py

```diff
@@ -0,0 +1,250 @@
+#!/usr/bin/python3
+"""Plot the spike-ins measured abundance for samples quality control.
+
+Scatter plot of measured abundance (in counts) of ERCC transcripts in
+log2 scale and their known concentration  [attomoles/ul] in the sample
+in log2 scale. The number of undetected ERCC transcripts are noted in
+the legend and are marked with gray points in the scatter plot. The
+coefficient of determination (R-squared value) is calculated by
+comparing  measured abundance of ERCCs in the sample with  known
+concentration of ERCC spike-ins.
+
+The histogram above the scatter plot, shows the number of ERCCs per
+input concentration. Missing ERCC transcripts are colored in gray.
+
+The scatter plot helps to infer the following:
+
+    - How well the RNA-seq experiment has measured the known
+      concentration of the spike-ins. The strength of this linear
+      relationship is described with coefficient of determination,
+      which is the fraction of variation in the dependent variable
+      (measured abundance) that is predictable from the independent
+      variable (input concentration). In general, values above 0.9 for
+      example are considered ideal.
+
+    - Whether the data spans the expected dynamic range (ERCC
+      abundances span a 2^20 dynamic range). Based on the limit of
+      detection for the experiment, one can identify and omit any
+      endogenous genes in the sample that are insufficiently expressed
+      for reliable analysis.
+"""
+
+import argparse
+
+import matplotlib
+import numpy as np
+import pandas as pd
+from matplotlib import pyplot as plt
+from resolwe_runtime_utils import error, warning
+from scipy import stats
+
+import utils
+
+# File with known ERCC concetrations
+ERCC_CONC_FILE = '/opt/resolwebio/assets/ERCC_table.txt'
+
+# known SIRV concetrations
+SIRV_CONC = {
+    'SIRV1': 8000,
+    'SIRV2': 6000,
+    'SIRV3': 11000,
+    'SIRV4': 7000,
+    'SIRV5': 12000,
+    'SIRV6': 18000,
+    'SIRV7': 7000,
+}
+
+ERCC_MIXES = {
+    "ercc_mix1": {
```

Comment:
We use double quotes for strings that are printed to users, so we may want to use single quotes here.